sw/host/tests/chip/usb/usb.rs

```diff
@@ -41,7 +41,7 @@ pub struct UsbOpts {
     pub strapping: Vec<String>,
```

Comment:
I would squash this commit, the line changed was intruduced by the previous commit.